src/quo/components/button/view.cljs

```diff
@@ -81,16 +82,21 @@
       (when before
         [rn/view
          [icons/icon before {:color icon-color}]])
-      [rn/view {:style (content-style type)}
+      (when loading
+        [rn/view {:style {:position :absolute}}
+         [rn/activity-indicator]])
+      [rn/view {:style (merge (content-style type)
+                              (when loading
+                                {:opacity 0}))}
        (cond
          (= type :icon)
          [icons/icon icon {:color icon-color}]
 
-         (string? children)
+         (or (keyword? children) (string? children))
          [text/text {:weight          :medium
                      :number-of-lines 1
                      :style           {:color text-color}}
-          children]
+          (utils.label/stringify children)]
```

Comment:
We want to have Quo as an independent library, but this couples the component hardly to the current status implementation of translations. If at some time we will want to separate the lib, or change the implementation of i18n then we will need to refactor Quo for that.